### PR TITLE
Add OWNERS files for console related projects

### PR DESCRIPTION
**Pull request by sjanota**
**Description**

Add OWNERS for:
- console-backend-service, its tests and chart
- console chart
- log-ui chart
- servicecatalog-ui chart
- lambda-ui chart
- microfrontend client
- idpresets client
- compass cockpit chart
- addons-ui chart

Also add kwiatekus as root owner

**Related issue(s)**

### CODE REVIEW

**Comment by derberg** [comment]
@sjanota Please update the description and make proper commit message and title of the PR as the current has no meaning for the community.

small comment regarding `console-approvers`, you use it in `reviewers` section too, please maintain 2 aliases as it is here already https://github.com/kyma-project/kyma/blob/master/OWNERS_ALIASES

File: components/console-backend-service/OWNERS [hunk 1]
```diff
@@ -0,0 +1,11 @@
+reviewers:
+  - console-approvers
+  - michal-hudy
```
Comment (michal-hudy) [inline components/console-backend-service/OWNERS]: As I rarely contribute to this project, please remove me from reviewers.

File: tests/console-backend-service/OWNERS [hunk 2]
```diff
@@ -0,0 +1,11 @@
+reviewers:
+  - console-approvers
+  - michal-hudy
```
Comment (michal-hudy) [inline tests/console-backend-service/OWNERS]: As I rarely contribute to this project, please remove me from reviewers.

File: components/console-backend-service/OWNERS [hunk 3]
```diff
@@ -0,0 +1,11 @@
+reviewers:
+  - console-approvers
+  - michal-hudy 
+  - m00g3n
+  - aerfio
```
Comment (aerfio) [inline components/console-backend-service/OWNERS]: I have made max 2 PRs in CBS in total, so there's no sense for me to be here, I wouldn't be able to help 😢 
Please remove me too

File: tests/console-backend-service/OWNERS [hunk 4]
```diff
@@ -0,0 +1,11 @@
+reviewers:
+  - console-approvers
+  - michal-hudy 
+  - m00g3n
+  - aerfio
```
Comment (aerfio) [inline tests/console-backend-service/OWNERS]: I have made max 2 PRs in CBS in total, so there's no sense for me to be here, I wouldn't be able to help 😢 
Please remove me too

File: components/console-backend-service/OWNERS (hunk 1 above)
Comment (sjanota) [reply components/console-backend-service/OWNERS]: done

File: tests/console-backend-service/OWNERS (hunk 2 above)
Comment (sjanota) [reply tests/console-backend-service/OWNERS]: done

File: components/console-backend-service/OWNERS (hunk 3 above)
Comment (sjanota) [reply components/console-backend-service/OWNERS]: done

File: tests/console-backend-service/OWNERS (hunk 4 above)
Comment (sjanota) [reply tests/console-backend-service/OWNERS]: done

**Comment by sjanota** [comment]
@derberg Is it better now?

**Comment by derberg** [comment]
@sjanota if you're asking :) I would say your commit adds files and doesn't create them, so I would start the message with `Add` but what you have now is already so much better than before that I don't complain

File: tests/console-backend-service/OWNERS [hunk 5]
```diff
@@ -0,0 +1,9 @@
+reviewers:
+  - console-reviewers
+  - m00g3n
```
Comment (derberg) [inline tests/console-backend-service/OWNERS]: Marcin (`m00g3n`) is out atm but I'm sure he is sharing the same point of view that Mateusz and Michal so please remove him as well, and `magicmatatjahu` too

File: components/console-backend-service/OWNERS [hunk 6]
```diff
@@ -0,0 +1,9 @@
+reviewers:
+  - console-reviewers
+  - m00g3n
```
Comment (derberg) [inline components/console-backend-service/OWNERS]: Marcin (`m00g3n`) is out atm but I'm sure he is sharing the same point of view that Mateusz and Michal so please remove him as well, and `magicmatatjahu` too

**Pull request merged**